deploy/kubernetes/webhook.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: webhook
```

Comment:
it's my understanding that app should be tracee in this case, as this is another component in the same app, wdyt?